blog/source/_posts/WhyFbg.md

```diff
@@ -0,0 +1,21 @@
+
+So if there are so many board game implementations out there, how would someone in their sane mind think that [yet another board game project would help ?](https://xkcd.com/927/) 
+
+![XKCD](https://imgs.xkcd.com/comics/standards.png)
+
+Our bet is that FreeBoardGame.org is different: While most implementations out there are closed source, we are a Free software project. That means that instead of competing with each other in creating yet another game, we are going to slowly but surely share our work and create games that can be played, modified, improved, studied or even forked by anyone.
+
+With that said, we are also not the first board game FOSS project. Even though there are several FOSS board game projects out there, most of them are one-off projects that implement one single game. An exception is [boardgame.io](https://boardgame.io), a framework that makes it easy to code turn based games. This project created a large community around it, and FreeBoardGame.org is built on top of it.
+
+In fact, in more practical terms, you can think FreeBoardGame.org being a publishing platform for your boardgame.io games. While implementing the game requires a good amount of work, so does delivering a good experience for thousands of users, and this will be our focus. This includes helping with the development and curating games that get into the platform, fostering the community of users, making sure that the website is fast and optimized, making sure it is discoverable, etc.
+
+Being a free (as in freedom) software project does not only mean that we will provide the source code to anyone, but also that we will have an open governance. Hence, we will always make sure to give credits where credits are due (i.e. credits for all games developed by a contributor). And if we ever make any revenue from the website, these funds will be spent as transparently as possible: prioritizing maintaining the servers, investing in developing more games, improving current games, developing the infrastructure further etc. This way, we increase our odds of having a sustainable project and you can be sure that all your work will go towards the community.
+
+Therefore, together we can join all these one-off efforts into a single, coherent community effort. You can have fun implementing some game you really like, or you can help building the infra-structure for several games at once. Either way, your effort will be multiplied by the effort of others in the community, as your game and contributions can reach thousands of users easily. Let's have fun together :).
```

Comment:
I think infrastructure is one word. I would also change "some game" to "a game".  "Your effort" to "Your efforts".

Reply:
Done.